operators/operator_sdk/osdk-migrating-to-v0-1-0.adoc

```diff
@@ -13,7 +13,7 @@ The recommended method for migrating your project is to:
 . Copy your code into the new project.
 . Modify the new project as described for v0.1.0.
 
-This guide uses the `memcached-operator`, the example project from xref:osdk-getting-started.adoc#osdk-getting-started[Getting started with the Operator SDK], to illustrate the migration steps. See the link:https://github.com/operator-framework/operator-sdk-samples/tree/aa15bd278eec0959595e0a0a7282a26055d7f9d6/memcached-operator[v0.0.7 memcached-operator] and link:https://github.com/operator-framework/operator-sdk-samples/tree/4c6934448684a6953ece4d3d9f3f77494b1c125e/memcached-operator[v0.1.0 memcached-operator] project structures for pre- and post-migration examples, respectively.
+This guide uses the `memcached-operator`, the example project from xref:osdk-getting-started.adoc#osdk-golang[Getting started with the Operator SDK], to illustrate the migration steps. See the link:https://github.com/operator-framework/operator-sdk-samples/tree/aa15bd278eec0959595e0a0a7282a26055d7f9d6/memcached-operator[v0.0.7 memcached-operator] and link:https://github.com/operator-framework/operator-sdk-samples/tree/4c6934448684a6953ece4d3d9f3f77494b1c125e/memcached-operator[v0.1.0 memcached-operator] project structures for pre- and post-migration examples, respectively.
```

Comment:
Same question about GitHub links (it's existing content, so I don't think it's an issue).